spectral_cube/analysis_utilities.py

```diff
@@ -341,39 +344,82 @@ def stack_cube(cube, linelist, vmin, vmax, average=np.nanmean, convolve_beam=Non
         If the cube is a VaryingResolutionSpectralCube, a convolution beam is
```

Comment:
Add extra note here that this will trigger a cube convolution. Or maybe a note in the general description above "Parameters"?